### PR TITLE
#53 implement snackbar

### PR DESCRIPTION
**Pull request by BorchBe99**
This implementation provides an Interface for a fully consumed Snackbar using notistack.

### CODE REVIEW

**Review by christophtrybek — CHANGES_REQUESTED** [review]
Please check indentation it should only consist of two spaces. Consider using .editorconfig trough an appropriate plugin in your IDE

**Comment by BorchBe99** [comment]
I fixed the indentation.

File: client/src/ts/global/CustomSnackbar.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/**
+ * Component that implements a custom Snackbar using notistack
+ */
+import React from "react";
+import { useSnackbar, SnackbarMessage, OptionsObject } from "notistack";
+import IconButton, { IconButtonProps } from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+import { SvgIconProps } from "@material-ui/core/SvgIcon";
+
+interface Props {
```
Comment (LennartRukower) [inline client/src/ts/global/CustomSnackbar.tsx:10]: Please add a comment to this interface.

File: client/src/ts/global/CustomSnackbar.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  snackbarMessage: SnackbarMessage;
+  snackProps?: OptionsObject;
+  buttonProps?: IconButtonProps;
+  iconProps?: SvgIconProps;
+  showID?: boolean;
+}
+const CustomSnackbar = (props: Props) => {
```
Comment (LennartRukower) [inline client/src/ts/global/CustomSnackbar.tsx:17]: Please add a comment to this component.

**Review by LennartRukower — APPROVED** [review]
Good job Benji!

File: client/src/ts/global/CustomSnackbar.tsx (hunk 1 above)
Comment (BorchBe99) [reply client/src/ts/global/CustomSnackbar.tsx:13]: Fixed

File: client/src/ts/global/CustomSnackbar.tsx (hunk 2 above)
Comment (BorchBe99) [reply client/src/ts/global/CustomSnackbar.tsx:28]: Fixed

File: client/src/ts/global/NotFound.tsx [hunk 3]
```diff
@@ -10,81 +10,82 @@ import CardContent from "@material-ui/core/CardContent";
...
-        marginLeft: "auto",
-        marginRight: "auto",
-      },
-    },
-    backButton: {
-      color: "white",
-      backgroundColor: "#f6891f",
-    },
-  }));
+    createStyles({
```
Comment (christophtrybek) [inline client/src/ts/global/NotFound.tsx:13]: Please use spaces instead of tabs

**Review by christophtrybek — CHANGES_REQUESTED** [review]
Issue with spaces vs tabs still persists

**Review by christophtrybek — APPROVED** [review]
Good job

**Pull request merged**